rust/worker/chroma_config.yaml

```diff
@@ -60,6 +60,7 @@ query_service:
         hnsw_cache_config:
             weighted_lru:
                 capacity: 8589934592 # 8GB
+        num_parallel_collections: 16
```

Comment:
Need better name for this

Reply:
its the number of parallel disk loads right?

Reply:
we create `num_parallel_collections*num_parallel_collections` partitions in the partitioned mutex structure. The intuition for square of this was the birthday paradox suggested by Robert

Reply:
Please comment things like that